runtime/drivers/duckdb/config.go

```diff
@@ -104,7 +103,7 @@ func newConfig(cfgMap map[string]any) (*config, error) {
 	if cfg.ThreadsOverride != 0 {
 		threads = cfg.ThreadsOverride
 	} else if cfg.CPU > 0 {
-		threads = int(cpuThreadRatio * float64(cfg.CPU))
+		threads = cfg.CPU
 		if threads <= 0 {
```

Comment:
Can also remove this check fractional CPU is no longer possible now which would get translated to 0 previously ?

Reply:
True - removed